lib/classes/Swift/Plugins/DefaultSenderPlugin.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/*
+ * This file is part of SwiftMailer.
+ * (c) 2009 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Sets the sender of a message if none is set
+ *
+ * @author     Adam Zielinski
+ */
+class Swift_Plugins_DefaultSenderPlugin implements Swift_Events_SendListener
+{
+
+    /**
+     * The default sender email.
+     *
+     * @var string
+     */
+    private $defaultSenderEmail;
+
+    /**
+     * The default sender name.
+     *
+     * @var string
+     */
+    private $defaultSenderName;
+
+    /**
+     * List if IDs of handled messages
+     *
+     * @var string[]
+     */
+    private $handledMessageIds = array();
+
+    /**
+     * Create a new ImpersonatePlugin to impersonate $sender.
```

Comment:
oops in "Conversation" tab it looked like you commented on phpdoc of properties so I improved documentation of properties and missed the constructor comment. Now it's fixed